src/test/java/net/kaczmarzyk/spring/data/jpa/utils/JsonBodyParamsTest.java

```diff
@@ -154,6 +161,21 @@ public void returnsEmptyValueFromJsonWhenKeyNotPresent() {
 		Assert.assertTrue(result.isEmpty());
 	}
 
+	@Test
+	public void returnsEmptyValueWhenRequestBodyIsNull() {
+		//given
+		String json = "{ \"key1\": { \"key2\": \"value\" }}";
+		MockedStatic<JsonParser> jsonParserMockedStatic = mockStatic(JsonParser.class);
+
+		//when
+		jsonParserMockedStatic.when((MockedStatic.Verification) JsonParser.parseString(json)).thenReturn(null);
```

Comment:
is there a case when JsonParser returns plain null? i tested parser for null argument and it throws NPE and for empty string value it returns JsonNull object